README.md

```diff
@@ -1,6 +1,6 @@
 # Membership Attribute Service
 
-The membership attribute service provides an API for managing and retrieving membership attributes associated with a user. It runs on https://members-data-api.theguardian.com/
+The membership attribute service provides an API for managing and retrieving supporter attributes associated with a user. It runs on https://members-data-api.theguardian.com/
 
```

Comment:
this one was a mistake.. should we keep this linet as it was or should we change it (possibly to something else)

Reply:
I think this is OK